scripts/cron.lua

```diff
@@ -0,0 +1,41 @@
+function compute(process, message, opts)
+    -- Early return when no body is provided
+    -- This handles the initial invocation during process setup
+    if not message.body or not message.body.body then
+        return process
+    end
+    
```

Comment:
Is message.body.body a specific spec, `path`, `task_id`, etc

May want to validate that the table meets the spec and each property on the table is the correct type